cmd/fossa/cmd/test/test.go

```diff
@@ -18,8 +18,20 @@ import (
 	"github.com/fossas/fossa-cli/config"
 )
```

Comment:
I ran `fossa test` on a project that returned just:
```
Test Failed. 67 issues found:

```
Apparently it couldn't loop over the issues to display them. I'll investigate and let you know what I find out.

Reply:
It's because we can end up with `issues` in func `Run` looking like:
```
{
  Count: 67,
  Issues: nil,
  Status: "SCANNED",
  NormalizedByType: nil,
}
```

The original API response which gets unmarshalled into an `Issues` struct:
```
{"count":67,"status":"SCANNED"}
```

Reply:
If you're using a push only API key, we don't return any issue data, only pass/fail. This is to avoid leaking information if a key is compromised.

Reply:
I see. The CLI should document this somehow (explain that displaying the results requires a full-access key). And if the user still provides a push-only key—and most users still will at least initially—then the output shown should say something like:
```
Test Failed. 67 issues found. Cannot retrieve issues with a push-only API key.
```

Reply:
Agreed, this will be addressed with issue #344 which is the next thing I'm working on.